### ISSUE TITLE
Part Templates

### ISSUE THREAD

**Issue by richardiphillips** [linn/purchasing#353]

**Comment by RossStewart** [comment]
Yeah, it seems pretty functional. Are there any changes required? Should this issue be closed if it's fine?

**Comment by richardiphillips** [comment]
Nice one. Seems like Jacki doesn't know it was there and it wasn't taken out of the menu.
